content/product/product_areas.md

```diff
@@ -0,0 +1,81 @@
+# Sourcegraph product areas
```

Comment:
@malomarrec this came in due to an issue with .gitignore I think, you should remove it from your PR.